package.json

```diff
@@ -3,18 +3,34 @@
   "version": "1.0.0",
   "description": "<a name=\"readme-top\"></a>",
   "main": "index.js",
+  "type": "module",
```

Comment:
You did great work but when I try to run `npm start`, I'm getting an error, and I can't run the project. you may use this [article](https://stackoverflow.com/questions/69099763/referenceerror-require-is-not-defined-in-es-module-scope-you-can-use-import-in) to fix this. Kindly fix it 😃   

![error](https://user-images.githubusercontent.com/57408419/227921727-8e90a209-2edc-471b-b2fe-a1e2114b4538.JPG)